### DISCUSSION TITLE
Is the model undertrained ?

### DISCUSSION THREAD

**Discussion by axel578** [PixArt-alpha/PixArt-alpha#66] [General]
It seems to not be aware of many concepts and compositions, I get nearly same generation for seed over seed.
Here is a demo to prove my point :
![image](https://github.com/PixArt-alpha/PixArt-alpha/assets/44734791/1c1dc81a-e9e1-46e2-ae65-55a88fcf8177)
![image](https://github.com/PixArt-alpha/PixArt-alpha/assets/44734791/a8035093-eb0a-4979-85cc-67dbd1158ff8)
![image](https://github.com/PixArt-alpha/PixArt-alpha/assets/44734791/0273e439-f90f-4c70-8ce6-8c1346d272e7)

I think more training could be really beneficial for this model since in terms of image quality and speed it has a lot of potential !

**Comment by lawrence-cj** [comment]
New version is already training. Will be better.
